apollo-router/resources/index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="utf-8" />
+    <link rel="icon" href="https://apollo-server-landing-page.cdn.apollographql.com/_latest/assets/favicon.png" />
+    <meta name="viewport" content="width=device-width,initial-scale=1" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link href="https://fonts.googleapis.com/css2?family=Source+Sans+Pro&display=swap" rel="stylesheet" />
+    <meta name="theme-color" content="#000000" />
+    <meta name="description" content="Apollo server landing page" />
+    <link rel="apple-touch-icon"
+        href="https://apollo-server-landing-page.cdn.apollographql.com/_latest/assets/favicon.png" />
+    <link rel="manifest" href="https://apollo-server-landing-page.cdn.apollographql.com/_latest/manifest.json" />
+    <title>Apollo Server</title>
+</head>
+
+<body style="margin: 0; overflow-x: hidden; overflow-y: hidden">
+    <noscript>You need to enable JavaScript to run this app.</noscript>
+    <div id="react-root">
+        <style>
+            .fallback {
+                opacity: 0;
+                animation: fadeIn 1s 1s;
+                animation-iteration-count: 1;
+                animation-fill-mode: forwards;
+                padding: 1em;
+            }
+
+            @keyframes fadeIn {
+                0% {
+                    opacity: 0;
+                }
+
+                100% {
+                    opacity: 1;
+                }
+            }
+        </style>
+        <div class="fallback">
+            <h1>Welcome to Apollo Server</h1>
+            <p>It appears that you might be offline. POST to this endpoint to query your graph:</p>
+            <code style="white-space: pre;">
+curl --request POST \\
+  --header 'content-type: application/json' \\
+  --url '<script>document.write(window.location.href)</script>' \\
+  --data '{"query":"query { __typename }"}'</code>
```

Comment:
We ... need to add this to our list of supported introspection queries. 🤣   (@o0Ignition0o can help with that if you're curious.)

Reply:
I added a new supported introspection query. I hope it's the right place @o0Ignition0o

Reply:
Ok I have good and bad news for you, good news is you added it in the right place, the bad news is ... we're matching exact queries so you want to "save without formatting" (you have `\n` s afaict)

Reply:
Should be good now